test/unit/services/test-authorisation.js

```diff
@@ -28,124 +28,102 @@ describe('services/authorisation', function () {
     })
     it('should throw a 403 error if user is not ' + applicationRoles.BAND_9, function () {
       try {
-        authorisation.hasRoles({ user: { roles: [applicationRoles.CLAIM_ENTRY_BAND_2, applicationRoles.CLAIM_PAYMENT_BAND_3, applicationRoles.CASEWORK_MANAGER_BAND_5, applicationRoles.APPLICATION_DEVELOPER, applicationRoles.HWPV_SSCL] } }, [applicationRoles.BAND_9])
+        authorisation.hasRoles({ user: { roles: [applicationRoles.CLAIM_ENTRY_BAND_2, applicationRoles.CLAIM_PAYMENT_BAND_3, applicationRoles.CASEWORK_MANAGER_BAND_5, applicationRoles.HWPV_SSCL] } }, [applicationRoles.BAND_9])
```

Comment:
Could the roles not have been destructed? Allot of `applicationRoles.` and I think the noise outweighs the context?

Reply:
This is not my code. Not fixing things like this as we want to sites to go live. Just wanting to remove an unused role.